File: src/scrambler.js
var numMoves = 25,
    moves = [],
    turnTypes = [ "U", "R", "L", "D", "B", "F" ],
    revTypes = [ " ", "2", "'" ],
    type = turnTypes[Math.floor(Math.random()*6)];
console.log();
while(moves.length < numMoves)
  if(moves == [] || moves.slice(-1)[0] != (type = turnTypes[Math.floor(Math.random()*6)])) {
    moves.push(type);
    process.stdout.write(type + revTypes[Math.floor(Math.random()*3)] + (moves.length % 5 == 0 ? "\n" : "\t"));
  }
console.log();

File: bin/scrambler.py
from random import randint

numMoves, moves = 25, []
turnTypes = [ "U", "R", "L", "D", "B", "F" ]
revTypes = [ " ", "2", "'" ]
print();
while len(moves) < numMoves:
  type = turnTypes[randint(0, 5)]
  if len(moves) == 0 or moves[-1] != type:
    moves.append(type)
    print("%s%s" % (type, revTypes[randint(0, 2)]), end = ("\n" if len(moves) % 5 == 0 else "\t"))
print();
